neuroanalysis/data/loaders.py

```diff
@@ -0,0 +1,480 @@
+import h5py
```

Comment:
This looks good, but I wonder if it could be made a bit more flexible in terms of the kinds of things that can be customized via loaders.. ideally any data model attribute could be either supplied at object initialization or dynamically requested from the loader (but I am not sure that making it so flexible would be worth the added infrastructural complexity)